src/js/worklets/header_animations.js

```diff
@@ -0,0 +1,23 @@
+import {
+  useDerivedValue,
+  interpolate,
+  interpolateColor,
+  Extrapolation,
+  useAnimatedStyle,
+} from 'react-native-reanimated';
+
+export function blurAmount(sharedValue) {
+  return useDerivedValue(() => {
+    return parseInt(interpolate(sharedValue.value, [0, 60], [1, 15], Extrapolation.CLAMP));
+  });
+}
+
+export function layerOpacity(sharedValue, from, to) {
```

Comment:
In that case this could be:


```
export const layerOpacity = (sharedValue, fromColor, toColor) => 
  useAnimatedStyle(() => ({
    flex: 1, // Consider moving this to a higher context if applicable
    justifyContent: 'flex-end',
    backgroundColor: interpolateColor(
      sharedValue.value, 
      [CLAMP_MIN, CLAMP_MAX], 
      [fromColor, toColor], 
      'RGB'
    ),
  }));
```

Reply:
Aggreed with @FFFra, the styles that are not relevant to animations should stay in the cljs land

Reply:
@clauxx @FFFra 
I'd prefer to keep then in the JS side, so we don't need to transform them from cljs->js which is expensive.